buildpack/util.py

```diff
@@ -122,7 +122,11 @@ def download_and_unpack(url, destination, cache_dir="/tmp/downloads"):
             cached_location=cached_location, dest=destination
         )
     )
-    if file_name.endswith(".tar.gz") or file_name.endswith(".tgz"):
+    if (
```

Comment:
Be sure to check if the extensions are picked up in `_delete_other_versions` as well. If they're not, this will cause issues in application build caches on Cloud Foundry.

Reply:
It's a regex that does the work there, I don't think it needs major adjustment

Reply:
done. thanks i had missed this. i verified with two versions of a file wherein older one wasn't removed from cache before but now it does.